### ISSUE TITLE
[Bug]In Mongodb->for the query->After double curly brackets the user is allowed enter any text and when user run it it should throw an error response

### ISSUE THREAD

**Issue by RakshaKShetty** [appsmithorg/appsmith#5309]
## Description

[What happened]In Mongodb->For Query ->After double curly brackets the user is allowed enter any text and when user run it it should throw an error response

Steps
1.Create a mongo db Datasource 
2.create a new query 
3.Select command as fins documents 
4.In query type {}abcd
5.click on run
6.verify the display

Actual result : the query run successfully without an error

https://drive.google.com/file/d/1Yeo3YqhQd3aAl9LZ9WWlW7f1BGYob0ye/view

Important Details
Version: [Cloud ]
OS: Windows
Browser Chrome
Environment Production

**Comment by trishaanand** [comment]
This is correct and expected behaviour. This is equivalent of not adding any conditions and just fetching all the data in the table/collection
